samples/peripheral/radio_test/sysbuild.cmake

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+get_property(PM_DOMAINS GLOBAL PROPERTY PM_DOMAINS)
+
+# Include app core image if enabled
+if(SB_CONFIG_BOARD_NRF5340DK_NRF5340_CPUNET)
```

Comment:
CPUNET or CPUAPP?

Reply:
CPUNET, because if you build on nrf5340_cpunet the CPUNET board is passed to sysbuild.